### ISSUE TITLE
Cancer subtyping procedure

### ISSUE THREAD

**Issue by davidhunt4** [prov-gigapath/prov-gigapath#27]
Hello, thank you all for the amazing work. I am wondering what steps were taken for cancer subtyping after tile- and slide-level embeddings were generated. Was a separate classification model implemented?
I am new to this field so please excuse if I missed this answer anywhere.

**Comment by HanwenXuTHU** [comment]
Hi no worries. For now, we released the base model instead of the fine-tuned subtyping model. But you can still make use of our codebase to get a subtyping model. Here are some steps.

First, please set up a new csv file following the style of [PANDA](https://github.com/prov-gigapath/prov-gigapath/blob/main/dataset_csv/PANDA/PANDA.csv) task. Second, please replace the slide ids and labels with the subtyping data you have. Third, define a new subtyping yaml file as [PANDA](https://github.com/prov-gigapath/prov-gigapath/blob/main/finetune/task_configs/panda.yaml). Finally, by setting up the fine-tuning script and kicking off the job, the slide encoder would be adapted to subtyping tasks.

**Comment by davidhunt4** [comment]
Thank you. When you say setting up the fine-tuning script, would this mean making a new .sh file for my dataset?

**Comment by HanwenXuTHU** [comment]
Yes, please set up a new sh file following the PANDA task.

**Comment by HanwenXuTHU** [comment]
Close this issue for now. Feel free to re-open it if you have further question!

**Comment by davidhunt4** [comment]
Hi, I have a new issue. I have followed your instructions and made a new .yaml file, new .sh file, and new .csv file adapted to my dataset. I have put h5 files with 'features' and 'coords' in a directory called NEW_h5 and I run the command
```shell
bash scripts/run_smab.sh data/NEW.h5
``` 
Everything seems to work well except it tells me that it is
```shell
Training on 3045 samples
Validating on 805 samples
Testing on 2681 samples
``` 
These are not the number of samples provided in the splits that I defined, and I do not know where these numbers come from. Do you have any insights?
